sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AdaptiveApplicationControlsOperations.cs

```diff
@@ -276,7 +280,7 @@ public async Task<AzureOperationResponse<AdaptiveApplicationControlGroups>> List
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<AdaptiveApplicationControlGroup>> GetWithHttpMessagesAsync(string groupName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<AdaptiveApplicationControlGroup>> GetWithHttpMessagesAsync(string ascLocation, string groupName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
public method signature change is breaking change, need to bump the major version